olm/README.md

```diff
@@ -0,0 +1,27 @@
+# Creating new OLM artifacts
+
+Script `update-nightly-olm-files.sh` is used for that
+
+Pre-requisites
+ - [yq/https://pypi.org/project/yq](https://pypi.org/project/yq) and not [http://mikefarah.github.io/yq/](http://mikefarah.github.io/yq/)
+ - [Operator SDK](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/install-operator-sdk.md)
+ 
+ in `olm` folder, execute `update-nightly-olm-files.sh` script
+ 
+ 
+It can be done in a docker environment:
+
+from the root cloned directory of che-operator:
+```bash
+$ cd git/eclipse/che-operator
+$ docker run --rm -it -v $(pwd):/workdir --entrypoint=sh evns/yq
+# now in docker context
+$ apk update
+$ apk add curl bash
+$ RELEASE_VERSION=v0.9.0
```

Comment:
Could you use version `v0.8.2` instead of `v9.0.0` (which changes the way it manages OLM CSV and package files) ?

Reply:
yes